src/app/favorite/components/favorite/favorite.component.html

```diff
@@ -0,0 +1,10 @@
+<div *ngIf="isFavorite; else elseBlock">
+  <mat-icon aria-hidden="false" (click)="onFavorite($event)">
+    favorite
+  </mat-icon>
+</div>
+<ng-template #elseBlock>
```

Comment:
give the template a name for what is inside, it can be use somewhere else apart from that specific else block